packages/manager/modules/pci/src/projects/project/storages/databases/database/namespaces/add/add.html

```diff
@@ -0,0 +1,294 @@
+<oui-back-button
+    data-previous-page="{{:: 'pci_databases_namespaces_add_back' | translate }}"
+    data-on-click="$ctrl.trackAndGoBack()"
+></oui-back-button>
+
+<h2 data-translate="pci_databases_namespaces_add_title"></h2>
+
+<form name="$ctrl.addNamespaceForm" ng-submit="$ctrl.add()">
+    <div class="row">
+        <div class="col">
+            <oui-field>
+                <label class="oui-field__label">
+                    <span
+                        data-translate="{{:: 'pci_databases_namespaces_add_name_label' | translate }}"
+                    ></span>
+                    <small class="text-danger">*</small>
```

Comment:
**question:** We should investigate around this behavior. Guidelines tend to only mark fields that are optional.

Reply:
Hi Antoine, can you link me the guidelines please ? I have found the same behavior in other pages in the manager:
`manager/apps/web/client/app/domain/dynhost/add/domain-dynhost-add.html:
manager/apps/web/client/app/domain/dynhost/edit/domain-dynhost-edit.html:
manager/apps/web/client/app/domain/dynhost/login/add/domain-dynhost-login-add.html:
manager/apps/web/client/app/domain/redirection/add/domain-redirection-add.html:
manager/apps/web/client/app/domain/redirection/edit/domain-redirection-edit.html:
manager/apps/web/client/app/domain/zone/record/header.html:
manager/apps/web/client/app/hosting/cron/add-or-edit/hosting-cron-add-or-edit.html:
manager/apps/web/client/app/hosting/envvars/add/hosting-envvars-add.html:
manager/apps/web/client/app/hosting/envvars/edit/hosting-envvars-edit.html:
manager/apps/web/client/app/hosting/module/add/hosting-module-add.html:
manager/apps/web/client/app/hosting/runtimes/add/hosting-runtimes-add.html:
manager/apps/web/client/app/hosting/runtimes/edit/hosting-runtimes-edit.html:
manager/apps/web/client/app/private-database/database/add/private-database-database-add.html:
manager/apps/web/client/app/private-database/user/add/private-database-user-add.html:
manager/apps/web/client/app/private-database/whitelist/add/private-database-whitelist-add.html:
manager/modules/email-domain/src/email/account/change-password/email-domain-email-account-change-password.html:
manager/modules/email-domain/src/email/account/create/email-domain-email-account-create.html:
manager/modules/email-domain/src/email/filter/create/email-domain-email-filter-create.html:
manager/modules/email-domain/src/email/filter/edit/email-domain-email-filter-edit.html:
manager/modules/email-domain/src/email/redirection/create/email-domain-email-redirection-create.html:
manager/modules/email-domain/src/email/redirection/update/email-domain-email-redirection-update.html:
manager/modules/email-domain/src/email/responder/create/email-domain-email-responder-create.html:
manager/modules/email-domain/src/email/responder/update/email-domain-email-responder-update.html:
manager/modules/email-domain/src/mailing-list/create/email-domain-mailing-list-create.html:
manager/modules/email-domain/src/mailing-list/update/email-domain-mailing-list-update.html:
manager/modules/emailpro/src/domain/update/emailpro-domain-update.html:
manager/modules/emailpro/src/external-contact/add/emailpro-external-contact-add.html:
manager/modules/emailpro/src/external-contact/update/emailpro-external-contact-update.html:
manager/modules/emailpro/src/mailing-list/create/emailpro-mailing-list-create.html:
manager/modules/emailpro/src/mailing-list/update/emailpro-mailing-list-update.html:
manager/modules/emailpro/src/redirection/create/email-domain-email-redirection-create.html:
manager/modules/emailpro/src/redirection/update/email-domain-email-redirection-update.html:
manager/modules/exchange/src/external-contact/update/external-contact-update.html:
manager/modules/exchange/src/group/add/group-add.html:
manager/modules/exchange/src/group/update/group-update.html:
manager/modules/exchange/src/shared/add/shared-add.html:
manager/modules/exchange/src/shared-account/add/shared-account-add.html:
manager/modules/exchange/src/shared-account/update/shared-account-update.html:
manager/modules/pci/src/projects/project/storages/databases/database/namespaces/add/add.html:
manager/modules/pci/src/projects/project/storages/databases/database/namespaces/edit/edit.html:`
This is how the page has been designed by CX, and is required in the specifications of the project, so I wont be able to change it

Reply:
Unfortunately most of the apps/modules listed here represent legacy parts except the `pci` modules.
For consistency across the control panel, this is definitely something that we need to take into in account.
Checking with the team the mockups and we'll keep you posted!

Reply:
Hi @antleblanc, according to the rules given by CX in the ticket comment, having the required fields here is normal.